aten/src/ATen/native/quantized/cpu/fbgemm_utils.h

```diff
@@ -16,6 +16,7 @@
 // Note that in JIT mode we can think of a way to fuse col_offsets with bias.
 struct FBGEMM_API PackedLinearWeight {
   std::unique_ptr<fbgemm::PackBMatrix<int8_t>> w;
+  c10::optional<at::Tensor> bias;
```

Comment:
Another option is to just store a std::vector<int32_t> bias instead of at::Tensor for bias. But the drawback is that we are changing the type of bias from int32_t to float, to resolve the scaling factor unknown issue for dynamic batching during the eager mode. cc @dskhudia . I am OK with the current design.